include/odp/api/spec/errno.h

```diff
@@ -65,6 +68,17 @@ void odp_errno_zero(void);
 */
 void odp_errno_print(const char *str);
 
+/**
+* Print ODP errno
+*
+* Interprets the value of ODP errno as an error message, and prints it,
+* optionally preceding it with the custom message specified in str.
```

Comment:
Instead of duplicating the documentation, perhaps it would make sense to write something like this:
`Like odp_errno_print(), but uses a callback function for output.`

Same goes for all the alt functions.

Reply:
Should it actually be "Provides the same information and format as odp_errno_print(), ...", or is that an unnecessary requirement?